### DISCUSSION TITLE
PR appeal

### DISCUSSION THREAD

**Discussion by ghost** [PrajaktaSathe/CPP#2] [General]
Hello.. I am new to open sourcing and wish to contribute.  Can I generate a PR?

**Comment by PrajaktaSathe** [comment]
You sure can contribute! If you want to add some code written in CPP that is not already present in the repository, you may open up a pull request. But please make sure that the code that you want to upload is written by you.

**Reply by ghost** [reply]
Okay. Thanks a lot.

**Comment by ghost** [comment]
[codes.zip](https://github.com/PrajaktaSathe/CPP/files/5765507/codes.zip)
Please have a look.

**Reply by PrajaktaSathe** [reply]
Hi!
Thanks for your interest in contributing to this repo, but please do not upload zip files. Please upload only your .cpp file as a pull request. You might want to refer [this](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests) website.
You might also want to refer [this](https://opensource.com/article/19/7/create-pull-request-github) website for how to open up a PR.
